### DISCUSSION TITLE
quick question with regards to email notifications

### DISCUSSION THREAD

**Discussion by skyeci-draytek** [alexjustesen/speedtest-tracker#1676] [Q&A]
hi, I hope someone can help. I have my email notifications working fine but on the notification the "view results" which links to the local host doesn't work. Can I edit the email template somewhere so it points to the right place?

thank you

**Comment by svenvg93** [comment]
Hi, 

you have to set the APP_URL environment variable to the correct address. This will then be used in the mails 🙂

**Comment by skyeci-draytek** [comment]
Thanks so much, worked first time :)

**Reply by svenvg93** [reply]
Cool! Happy it worked!
